bin.src/makeGeom.py

```diff
@@ -146,8 +143,8 @@ def printRaftCcdGeom(buff, ccdId, ccdname, xidx, yidx, xpos, ypos):
             buff.write('    serial: -1 \n')
 
     buff.write('    Ccd: { \n')
-    #buff.write('        name: "R:0,0 S:%d,%d CFHTid:%d %s" \n' % (xidx, yidx, ccdId, ccdname))
-    #buff.write('        serial: -1 \n')
+    # buff.write('        name: "R:0,0 S:%d,%d CFHTid:%d %s" \n' % (xidx, yidx, ccdId, ccdname))
+    # buff.write('        serial: -1 \n')
```

Comment:
Consider deleting commented-out code. I can see why you kept it -- better safe than sorry, especially as it's not your package. @PaulPrice what do you think?

Reply:
I think this file is left over from the cameraGeom transition half a lifetime ago, and it looks like it contains CFHT-specific stuff (?!). Suggest we simply dump it all.